packages/editor-ui/src/components/MainHeader/WorkflowDetails.vue

```diff
@@ -552,7 +551,7 @@ async function onWorkflowMenuSelect(action: WORKFLOW_MENU_ACTIONS): Promise<void
 			}
 
 			try {
-				await workflowsStore.deleteWorkflow(props.workflow.id);
+				await workflowsStore.deleteWorkflow(props.id);
```

Comment:
Can we just emit an event here so we can get rid of the store?
```suggestion
				emit('deleteWorkflow');
```

Reply:
[Follow-up ticket](https://linear.app/n8n/issue/PAY-1942/further-optimizations-on-workflowdetails-component)